src/Resources/views/Menu/sonata_menu.html.twig

```diff
@@ -47,4 +47,4 @@
     {% endapply %}
 {% endblock %}
 
-{% block label %}{% if is_link is defined and is_link %}{{ icon|default|raw }}{% endif %}{% if options.allow_safe_labels and item.extra('safe_label', false) %}{{ item.label|raw }}{% else %}{{ item.label|trans(item.extra('translation_params') | default({}), translation_domain|default('messages')) }}{% endif %}{% endblock %}
+{% block label %}{% if is_link is defined and is_link %}{{ icon|default|raw }}{% endif %}{% if options.allow_safe_labels and item.extra('safe_label', false) %}{{ item.label|raw }}{% else %}{{ item.label|trans(item.extra('label_translation_parameters')|default({}), translation_domain|default('messages')) }}{% endif %}{% endblock %}
```

Comment:
```suggestion
{% block label %}{% if is_link is defined and is_link %}{{ icon|default|raw }}{% endif %}{% if options.allow_safe_labels and item.extra('safe_label', false) %}{{ item.label|raw }}{% else %}{{ item.label|trans(item.extra('label_translation_parameters', {}), translation_domain|default('messages')) }}{% endif %}{% endblock %}
```

Reply:
Maybe this could be a good occasion to introduce some linebreaks. To make sure there is no space in the output, we could use Twig's [whitespace control features](https://twig.symfony.com/doc/2.x/templates.html#templates-whitespace-control).